src/compound_targets.py

```diff
@@ -16,8 +15,9 @@
 
 @mod.capture(
     rule=(
-        "[{user.cursorless_range_connective}] <user.cursorless_primitive_target> | "
-        "<user.cursorless_primitive_target> {user.cursorless_range_connective} <user.cursorless_primitive_target>"
+        "<user.cursorless_primitive_target> | "
+        "[<user.cursorless_range_type>] {user.cursorless_range_connective} <user.cursorless_primitive_target> | "
```

Comment:
So did we want to switch this one back to "slice past"?  Might make it clearer that it's not really a new scope type, but rather a different way of forming a range.  And then makes it quite distinct from "column", which will be a proper scope type

Reply:
Yes we can do that. Since it has the correct placement we can also use up/down if someone rather prefers that

Reply:
done